src/jmcomic/jm_client_impl.py

```
@@ -1032,13 +992,19 @@ def req_api_domain_server(self, url):
         else:
             return res_data['Server']
 
-    def update_api_domain(self):
-        if True is JmModuleConfig.FLAG_API_CLIENT_AUTO_UPDATE_DOMAIN_DONE:
-            return
+    def update_old_api_domain(self, new_server_list: list[str]):
+        if new_server_list and sorted(self.domain_list) == sorted(JmModuleConfig.DOMAIN_API_LIST):
```

Ruff (0.14.10)

`JmModuleConfig` may be undefined, or defined from star imports

(F405)